course/meetings/eeb-marty-2017-04-13.md

```diff
@@ -0,0 +1,59 @@
+---
+contact: Asher Cutter, Martin Krkosek
+department: Ecology and Evolutionary Biology
+uoftcoders: Madeleine, Luke, Joel, Elliott
+date: 2017-04-13
+---
+
+# Action Items
+- Madeleine: Add some specifics to the ecology topics in the syllabus
```

Comment:
I'll add Luke to the list and hopefully have something for you by the end of this week, although it might be early next week.